### PR TITLE
Fix flakiness in bf1 scenario

### PR DESCRIPTION
**Pull request by LefterisJP**
Fixes: #5078 

## Description

Check https://github.com/raiden-network/raiden/issues/5078#issuecomment-548380011

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Documentation
    - [ ] A new CLI flag was introduced, is there documentation that explains usage?
- Specs
    - [ ] If this is a protocol change, are the specs updated accordingly? If so, please link PR from the specs repo.
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

File: raiden/tests/scenarios/bf1_basic_functionality.yaml [hunk 1]
```diff
@@ -228,7 +230,7 @@ scenario:
           name: "Assert after 100 payments from 3 to 0"
           tasks:
             - wait: 100
-            - assert_sum: {from: 0, balance_sum: 1_995_000_000_000_000_000}
+            - assert_sum: {from: 0, balance_sum: 1_105_000_000_000_000_000}
```
Comment (czepluch) [inline raiden/tests/scenarios/bf1_basic_functionality.yaml]: What the reason for this change? If I remember correctly this scenarios has successfully run several times in the past, with the previous value?

Comment (LefterisJP) [reply raiden/tests/scenarios/bf1_basic_functionality.yaml]: Please read the linked issue and/or the commit messages. The scenario has not ran succesfully for weeks now.

TL;DR: One of the channels is closed and settled and as such its balance is not taken into account.

Comment (czepluch) [reply raiden/tests/scenarios/bf1_basic_functionality.yaml]: Yeah, just saw the issue comment afterwards. (For some reason I expected the reasoning of the PR to be explained in the PR itself and not the issue)

> The scenario has not ran succesfully for weeks now.
I am aware, but was just wondering why it ever passed in the first place then.

Comment (LefterisJP) [reply raiden/tests/scenarios/bf1_basic_functionality.yaml]: > I am aware, but was just wondering why it ever passed in the first place then.

Because the settlement timeout has gotten reduced to make scenario runs faster, in combination with something (no idea what yet) in Raiden making things go slower resulting in slow scenario runs.

Comment (christianbrb) [reply raiden/tests/scenarios/bf1_basic_functionality.yaml]: A PR description would have easier to reason about the change:

> Finally to make the scenario pass we wait for 40 blocks to make sure that the settlement timeout has passed and the channel0-4 is settled and as such taken out of the balance calculation. And we adjust the asserted balance to 1_105_000_000_000_000_000.

Comment (LefterisJP) [reply raiden/tests/scenarios/bf1_basic_functionality.yaml]: Read the issues or the commit messages which you copied pasted.
